packages/ui/src/components/va-app-bar/VaAppBar.stories.ts

```diff
@@ -1,12 +1,132 @@
-import { defineComponent } from 'vue'
-import VaAppBar from './VaAppBar.demo.vue'
+import { VaAppBar } from './'
 
 export default {
   title: 'VaAppBar',
   component: VaAppBar,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = {
+  args: {
+    default: 'App Bar',
+  },
+}
+
+export const Gradient = {
+  args: {
+    default: 'App Bar',
+    gradient: true,
+  },
+}
+
+export const Color = () => ({
+  components: { VaAppBar },
+  template: `
+    [warning]
+    <va-app-bar color="warning">
+      App Bar 
+    </va-app-bar>
+    [#ce6e67]
+    <va-app-bar color="#ce6e67">
+      App Bar 
+    </va-app-bar>
+  `,
+})
+
+export const Fixed = () => ({
+  components: { VaAppBar },
+  template: `
+    <va-app-bar fixed>
+      App Bar 
+    </va-app-bar>
+    {{ $vb.lorem(600) }}
+  `,
+})
+
+export const Bottom = () => ({
+  components: { VaAppBar },
+  template: `
+    <va-app-bar fixed bottom>
+      App Bar 
+    </va-app-bar>
+    {{ $vb.lorem(600) }}
+  `,
+})
+
+export const HideOnScroll = () => ({
+  components: { VaAppBar },
+  template: `
+    <div class="flex flex-col relative max-h-32 overflow-hidden">
+      <va-app-bar 
+        hide-on-scroll 
+        target="#target1"
+      >
+        App Bar 
+      </va-app-bar>
+      <div 
+        id="target1" 
+        class="pb-[500px] overflow-auto"
+      >
+        {{ $vb.lorem(2500) }}
+      </div>
+    </div>
+  `,
+})
+
+export const ShadowOnScroll = () => ({
+  components: { VaAppBar },
+  template: `
+    <div class="flex flex-col relative max-h-32 overflow-hidden">
+      <va-app-bar 
+        shadow-on-scroll 
+        target="#target2"
+      >
+        App Bar 
+      </va-app-bar>
+      <div 
+        id="target2" 
+        class="pb-[500px] overflow-auto"
+      >
+        {{ $vb.lorem(2500) }}
+      </div>
+    </div>
+  `,
+})
+
+export const ShadowColor = () => ({
```

Comment:
Can we scroll this by default somehow? I'd like visual test to capture shadow color.

Reply:
it can't be helped